<launch>

     <node name="crazyflie_velocity" pkg="iarc7_sensors"
     type="crazyflie_velocity.py"/>

</launch>